seed_data/lib.py

```diff
@@ -2,14 +2,13 @@
 Library functions for interacting with test data
```

Comment:
There are some references to `last_request` here which was deleted. I mentioned `CachedHandler` before but there's also `deserialize_user_data` and `CachedModelDeserializer`